src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -2018,7 +2020,14 @@ void DBusThreadObjectRcp::DeactivateEphemeralKeyModeHandler(DBusRequest &aReques
         error = OT_ERROR_FAILED;
```

Comment:
Why it is an error that border agent is stopped when trying to deactivate the ePSKc mode? Can't ePSKc be deactivated when border agent is stopped? I think this `DBusThreadObjectRcp::DeactivateEphemeralKeyModeHandler` should be implemented without checking the border agent state. For example:

```
void DBusThreadObjectRcp::DeactivateEphemeralKeyModeHandler(DBusRequest &aRequest)
{
    otError error        = OT_ERROR_NONE;
    auto    threadHelper = mHost.GetThreadHelper();
    bool    retain_active_session;
    auto    args = std::tie(retain_active_session);

    VerifyOrExit(mBorderAgent.GetEphemeralKeyEnabled(), error = OT_ERROR_NOT_CAPABLE);

    if ((DBusMessageToTuple(*aRequest.GetMessage(), args) == OTBR_ERROR_NONE) && !retain_active_session)
    {
        otBorderAgentDisconnect(threadHelper->GetInstance());
    }
    otBorderAgentClearEphemeralKey(threadHelper->GetInstance());

exit:
    aRequest.ReplyOtResult(error);
}
```

Reply:
Accepted. The key diffs are removing `OT_ERROR_FAILED` and `OT_ERROR_INVALID_STATE` to keep deactivate ePSKc successful. Add log for production debug.